habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -827,7 +911,7 @@ class TaskConfig(HabitatBaseConfig):
     # Spawn parameters
     physics_stability_steps: int = 1
     num_spawn_attempts: int = 200
-    spawn_max_dists_to_obj: float = 2.0
+    spawn_max_dists_to_obj: float = 1.0
```

Comment:
is this from object or from viewpoint?

Reply:
It is still obj in this PR. Will change this naming in the upcoming PR which spawns wrt viewpoints